src/controllers/change.answers.controller.ts

```diff
@@ -7,13 +7,13 @@ import {CHANGE_ANSWER_KEY} from "../constants";
 export const get = (req: Request, res: Response, next: NextFunction) => {
   const session: Session | undefined = req.session as Session;
   if (session) {
-    const query: string | undefined = req.query["changePage"] as string;
-    if (query) {
+    const changePageQuery: string | undefined = req.query["changePage"] as string;
```

Comment:
@mattch1 sorry to be picky, but is this really a 'query'? It looks to me like 'query' is a function we call on the request object in order to get the value of the 'changePage' path variable. So what's returned is not a query, just a value. I think just 'changePage' as a variable name would be less misleading.

Reply:
The value is the value of the query on the url e.g. url?changeName=startPage. The "query" conveys where the value has come from so it would no be less misleading to leave it out.

Reply:
Ok. Fair enough. Happy with the reasoning here. Will approve.